frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/active-health-checks.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+import { MINUTE } from '../consts';
+
+describe('Test Popover behaviour for different active health check cases.', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    cy.install();
+    cy.visit('/ocs-dashboards/block-file');
+  });
+
+  after(() => {
+    checkErrors();
+    cy.logout();
+  });
+
+  const scaleDownAndOpenHC = (state: string, deployments: string[]) => {
+    const scaleTo = (replicas: number) =>
+      deployments.forEach((deployment) => {
+        cy.exec(`oc scale --replicas=${replicas} deploy ${deployment} -n openshift-storage`);
+      });
+    scaleTo(0);
+    cy.byTestID(`${state}-secondary-status`, { timeout: 2 * MINUTE }).should('be.visible');
+    cy.byTestID('Storage Cluster-health-item')
+      .contains('Storage Cluster')
+      .click();
+    return () => scaleTo(1);
+  };
+
+  const verifyMessages = (expectedMessages: string[]) => {
+    cy.byTestID('healthcheck-message', { timeout: 2 * MINUTE })
+      .should('have.lengthOf', expectedMessages.length)
+      .each((row, index, collection) => {
+        const expectedMessage = expectedMessages[collection.length - index - 1];
+        const actualMessage = row.text();
+        expect(actualMessage).toEqual(expectedMessage);
+      });
+  };
+
+  it('Popover shows all warnings.', () => {
+    const restoreResources = scaleDownAndOpenHC('Warning', ['rook-ceph-mon-a', 'rook-ceph-mgr-a']);
```

Comment:
```suggestion
    const restoreResources = scaleDownAndOpenHC(State.Warning, [Deployments.ROOK_MON_A, Deployments.ROOK_MGR_A]);
```
Enumize these hardcoded strings.